documentation/API-onboarding.md

```diff
@@ -1,81 +1,237 @@
-- **Accepted** In this case, the TSC shall specify whether the API proposal is to be hosted by a new or existing Sub Project.
+The Sub-Project will follow the versioning guidelines defined by the Commonalities Working Group to maintain API versions. 
 
-Main tenets for TSC voting:
+When a new public release for an API is approved by the Release Management working group and created by the within the API Repository, the [CAMARA API overview list](https://github.com/camaraproject/APIBacklog/blob/main/documentation/APIbacklog.md) will be updated accordingly.
 
-- Tenet #1: Open-source projects are contribution-driven. There should be no guidelines to prevent partners to contribute. If a company would like to work on a certain API proposal they should be welcome to do it.
-- Tenet #2: The TSC shall make sure that APIs belonging together (close topic/content, API families) are developed within the same Sub Project, while ensuring that Sub Projects are not too small (difficult to track them down) or too big (difficult to be managed). There should be only one API for each purpose.
-- Tenet #3: There could be situations where API proposal complies with acceptance criteria (template duly filled out) but has not supporters other than the proponent company. What to do in this situation? Recommendation is to accept the API proposal and see how successful it will be in terms of industry adoption.
+### Documentation in the CAMARA API Overview List
 
-The TSC may also propose changes to an API proposal and take the decision considering these changes. The TSC documents the decision in the [CAMARA API overview list](https://github.com/camaraproject/APIBacklog/blob/main/documentation/APIbacklog.md) (fills in columns TSC date and TSC decision / Sub Project; in case of a No-Go "Rejected" is documented there).
+The **CAMARA API Overview List** is a centralized table used to document all API proposals, their progress, and metadata throughout their lifecycle. It provides transparency and traceability for all stakeholders involved in the CAMARA project.
 
-## API management and maintenance
+The overview list is structured with the following columns:
 
-In case of a GO decision of the TSC the named Sub Project takes over responsibility for the new API or API family and starts making contributions for the API definition(s), API documentation, provider implementation and test artefacts.
+| **Column Name**                                | **Description**                                                                                          |
+|------------------------------------------------|----------------------------------------------------------------------------------------------------------|
+| **API [Family] Name**                          | The name of the API or API family (e.g., "Messaging API Family").                                         |
```

Comment:
I would like to get rid of the "family" term already here in this version. There is (and was) no such thing as an API "family" in the project Governance. A proposal (and with that an API Repository) is about an API or potentially multiple APIs, which are closely related.

In addition is the should the term "API" or "APIs" never be part of the name.

```suggestion
| **API(s) Name**                          | The name of the API or API(s) (e.g., "Messaging" API(s)).                                         |
```